hedera-node/hedera-app/src/xtest/java/token/update/AbstractTokenUpdateXTest.java

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
This class is never used and could be deleted

Reply:
Removed